__tests__/app.test.js

```diff
@@ -27,4 +27,30 @@ describe ('app', () => {
             })
         })
     })
+    describe('GET /api', () => {
+        test('responds with an object containing all available endpoints', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+            .then((response) => {
+                const endPoints = response.body.endpoints
+                expect(endPoints).toHaveProperty('GET /api', expect.any(Object))
```

Comment:
as we add to the endpoints.json file every time we create a new endpoint, this test will become outdated - it wont test that all of the endpoints are coming back. therefore we should directly require in the endpoints.json file here and then check if the response body is equal to this file, as this file will always be up to date (json can be required in in the same way as javascript)